.github/workflows/release.yaml

```diff
@@ -53,6 +53,15 @@ jobs:
           version=$(echo -n $tag | sed 's/v//g')
```

Comment:
out of curiosity, why do we need to do a sed here?

Reply:
Because the downloaded source code file does not include the tag `v0.89.0` but the version `0.89.0`:
```
$ gh release download v0.89.0 -A tar.gz /tmp
$ ls -la /tmp/
total 38288
drwxrwxrwt  8 root     wheel       256 May 29 14:22 .
drwxr-xr-x  6 root     wheel       192 May 22 12:57 ..
-rw-r--r--@ 1 javirln  wheel  18898136 May 29 14:22 chainloop-0.89.0.tar.gz
```

Reply:
So, you can't use $tag for the container image tag do you?

Reply:
The container image tag does contain the tag: https://github.com/chainloop-dev/chainloop/pkgs/container/chainloop%2Fcontrol-plane/220543906?tag=v0.89.0

Reply:
My bad Javi, I was looking at the other `sed`. Sorry about the noise and thanks for the clarification.